tools/openblas_support.py

```diff
@@ -43,27 +43,26 @@ def download_openblas(target, arch, ilp64):
     fnsuffix = {None: "", "64_": "64_"}[ilp64]
     filename = ''
     if arch in ('aarch64', 'ppc64le', 's390x'):
-        filename = '{0}/openblas{1}-{2}-manylinux2014_{3}.tar.gz'.format(
-                        RACKSPACE, fnsuffix, OPENBLAS_LONG, arch)
+        filename = f'{RACKSPACE}/openblas{fnsuffix}-{OPENBLAS_LONG}-manylinux2014_{arch}'
```

Comment:
```suggestion
        filename = f'{RACKSPACE}/openblas{fnsuffix}-{OPENBLAS_LONG}-manylinux2014_{arch}.tar.gz'
```

Reply:
Or since the line length is getting long, perhaps
```suggestion
        suffix = f'manylinux2014_{arch}.tar.gz'
        filename = f'{RACKSPACE}/openblas{fnsuffix}-{OPENBLAS_LONG}-{suffix}'
```

Reply:
This should probably fix the CI failure